Cargo.toml

```diff
@@ -6,13 +6,13 @@ edition = "2018"
 
 [dependencies]
 # TODO: Limit the crates dependencies (mainly to reduce compile times)
-bevy = { path = "../bevy" }
+bevy={git="https://github.com/bevyengine/bevy", branch="main"}
 
 smallvec = "1.6.1"
 crossbeam = "0.8.0"
 
 [dev-dependencies] 
-bevy_flycam = { path = "../bevy_flycam" }
+# bevy_flycam = { path = "../bevy_flycam" }
```

Comment:
I commented that out because I just wanted a quick fix. This can probably be readded later.